public/index.pug

```diff
@@ -175,5 +174,7 @@ html(lang="en-US")
           title Twitter logo
           path(d=twitterIcon.path)
         span Share this on Twitter!
-
+      div(class="footer-repo")
+          p.
+            #[a(title="github repository" href="https://github.com/simple-icons/simple-icons-website") Made with &#10084; on GitHub]
```

Comment:
We should probably distinguish this `title` from [this one](https://github.com/simple-icons/simple-icons-website/blob/7843220e0a2b825dffc2e3f58fb9d41fae7e0e5b/public/index.pug#L32) more clearly, here's some suggestions:

- header: "project repository", "main repository", "icon repository"
- footer: "website repository", "website source code", "website source"

Reply:
true. Done.